packages/components/src/internal/components/chart/ChartFieldAggregateOptions.test.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
+import { ChartFieldAggregateOptions } from './ChartFieldAggregateOptions';
+import { BAR_CHART_AGGREGATE_NAME, BAR_CHART_ERROR_BAR_NAME } from "./constants";
+
+const field = { name: 'testField', label: 'Test Label', required: false };
+const fieldValues = {
+    testField: { value: 'ABC' },
+    [BAR_CHART_AGGREGATE_NAME]: { value: 'SUM' },
+    [BAR_CHART_ERROR_BAR_NAME]: undefined,
+};
+
+function renderComponent(props = {}) {
+    return render(
+        <ChartFieldAggregateOptions
+            field={field}
+            fieldValues={fieldValues}
+            includeCount={true}
+            includeNone={true}
+            onErrorBarChange={jest.fn}
+            onSelectFieldChange={jest.fn}
+            {...props}
+        />
+    );
+}
+
+describe('ChartFieldAggregateOptions', () => {
+    test('renders gear icon and overlay', async () => {
+        renderComponent();
+        expect(document.querySelectorAll('.field-option-icon')).toHaveLength(1);
+        expect(document.querySelectorAll('.fa-gear')).toHaveLength(1);
+        expect(document.querySelectorAll('.lk-popover')).toHaveLength(0);
+
+        await userEvent.click(document.querySelector('.fa-gear'));
+        expect(document.querySelectorAll('.lk-popover')).toHaveLength(1);
+    });
+
+    test('shows aggregate method select and error bar radio group in overlay', async () => {
+        renderComponent();
+        await userEvent.click(document.querySelector('.fa-gear'));
+        expect(document.querySelectorAll('label')[0].textContent).toContain('Aggregate Method');
+        expect(document.querySelectorAll('label')[1].textContent).toContain('Error Bars');
+        expect(document.querySelectorAll('.field-option-radio-group')).toHaveLength(1);
+        expect(document.querySelectorAll('.select-input-container')).toHaveLength(1);
+        expect(document.querySelectorAll('input[type="radio"]')).toHaveLength(3); // None, SD, SEM
```

Comment:
I think in the long term we'd be better off if we select the inputs we expect to be disabled, and assert they're disabled, and select the inputs we expect to be enabled, and assert they're enabled. Then these tests won't fail when we simply add more options to the UI, but instead will continue to work as intended. Then all it would take is adding new specific assertions for the pieces of UI that are added, instead of updating a count.